pipeline/load.py

```diff
@@ -3,9 +3,11 @@
 # pylint: disable=C0301
 # pylint: disable=E1101
 
+import os
```

Comment:
It seems you only use remove from the os package so instead of importing the entire os package every time you can use 
```suggestion
from os import remove 
```
and change the below line where you use os.remove to just remove